avm/res/security-insights/data-connector/tests/e2e/waf-aligned/dependencies.bicep

```
@@ -0,0 +1,73 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Workspace to create.')
+param name string
```

```suggestion
param logAnalyticsWorkspaceName string
```
-> Requires update in other places